### DISCUSSION TITLE
How can I find the time elapsed from the initial deployment of the application to when the health status turns to “healthy”?

### DISCUSSION THREAD

**Discussion by devagunay** [argoproj/argo-cd#20269] [Q&A]
Hello,

I want to find the time elapsed from applications first deployed (health status "Missing") to (health status turns to “healthy”) with using argocd cli commands.

After getting whole application information with "argocd app list" to app.json and investigate it. I cannot find any information for calculate this time.

There are two lines under operationState in app.json which are startedAt and finishedAt but this finishedAt time is give us the sync duration time.

Other than this finishedAt I can not find time value for calculate this. 

Do you have idea how we can calculate this elapse?

Thank you.

**Comment by alperpolat-meister** [comment]
Bump, this would be useful to know!
